ultralytics/nn/autobackend.py

```diff
@@ -393,7 +399,25 @@ def forward(self, im, augment=False, visualize=False, embed=None):
             y = self.session.run(self.output_names, {self.session.get_inputs()[0].name: im})
         elif self.xml:  # OpenVINO
             im = im.cpu().numpy()  # FP32
-            y = list(self.ov_compiled_model(im).values())
+
+            if self.inference_mode == "CUMULATIVE_THROUGHPUT":  # optimized for larger batch-sizes
```

Comment:
```suggestion
            if self.inference_mode != "THROUGHPUT":  # optimized for larger batch-sizes
```

Reply:
That change will cover THROUGHPUT and CUMULATIVE_THROUGHPUT, even the latter is recommended